website/validators.py

```diff
@@ -0,0 +1,10 @@
+from django.core.exceptions import ValidationError
+from django.conf import settings
+
+def validate_file_size(value):
+    filesize = value.size
+    
+    if filesize > settings.MAX_FILE_UPLOAD:
+        raise ValidationError("The maximum file size that can be uploaded is 4MB")
```

Comment:
this error message needs to use the constant

Reply:
Also maybe reword slightly, e.g. "The maximum file size for profile photos is " + constant